### PR TITLE
Postgres search by first/last name

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresQueryBuilder.java [hunk 1]
```diff
@@ -145,7 +147,10 @@ public void navnOgFodselsnummerSok(String soketekst) {
             whereStatement.add(FODSELSNR + " LIKE '" + soketekst + "%'");
         } else {
             String soketekstUpper = soketekst.toUpperCase();
-            whereStatement.add("(UPPER(" + FORNAVN + ") LIKE '%" + soketekstUpper + "%' OR UPPER(" + ETTERNAVN + ") LIKE '%" + soketekstUpper + "%')");
+            List<String> nameSplittedBySpaces = splitStringBySpaces(soketekstUpper);
```
Comment (stianStensli) [inline src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresQueryBuilder.java:150]: Hmm should we find a cleaver way to make this work for double names like FORNAVN = "Kari Anne" ?

Comment (stianStensli) [reply src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresQueryBuilder.java:150]: And also if you have time, I read that using UPPER on columns are really inefficient. Maybe we can just ensure that all names are always stored as uppercase in the DB?

Comment (dragutin-nav) [reply src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresQueryBuilder.java:150]: Hm, I am not sure that upper will influence query, I will test that assumption. I will try to write some tests for double name, just to see how current solution is performing :) Btw, thank you for useful comment!

File: src/test/java/no/nav/pto/veilarbportefolje/postgres/PostgresServiceTest.java [hunk 2]
```diff
@@ -53,36 +54,62 @@ public void setup() {
...
         Filtervalg teskt = new Filtervalg().setNavnEllerFnrQuery("test");
         Filtervalg fnr = new Filtervalg().setNavnEllerFnrQuery("123");
-        postgresService.hentBrukere("1234",null, null,null, teskt, 0, 10);
-        postgresService.hentBrukere("1234",null, null,null, fnr, 0, 10);
+        postgresService.hentBrukere("1234", null, null, null, teskt, 0, 10);
+        postgresService.hentBrukere("1234", null, null, null, fnr, 0, 10);
+    }
+
+    @Test
+    public void sok_pa_navn_case_insesitive() {
```
Comment (dragutin-nav) [inline src/test/java/no/nav/pto/veilarbportefolje/postgres/PostgresServiceTest.java:83]: @stianStensli can you please check tests, I am not sure what kind of scenarios to test beside those

**Comment by stianStensli** [comment]
Lukker da vi nå skal bruke Opensearch

**Pull request closed without merging**